### PR TITLE
PMM-7153 Release Notes 2.13.0

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

File: docs/release-notes/2.13.0.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    Improves the user experience for adding and viewing external services on the Node Summary dashboard of PMM. External services means any data that can be monitored by a Prometheus export, for example, non-Percona supported databases like Reddis, ElasticSearch, Cassandra, etc. or an organization’s external application.
+
+4. DBaaS Preview phase 1.0 (Technical Preview). We are also releasing the first preview of DBaaS functionality. You can read more about [configuration](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/dbaas.html) and [usage](https://www.percona.com/doc/percona-monitoring-and-management/2.x/using/platform/dbaas.html).
+
+
+
+## Improvements
+
+* [PMM-5364](https://jira.percona.com/browse/PMM-5364): Ability to monitor SSL-enabled MongoDB (Thanks to Hubertus Krogmann for reporting this issue)
+* [PMM-7086](https://jira.percona.com/browse/PMM-7086): VictoriaMetrics end-points replace Prometheus GUI in PMM Server 2.12.0 (Thanks to Daniel Guzman Burgos for reporting this issue)
```
Comment (shoffman-percona) [inline docs/release-notes/2.13.0.md:50]:
```suggestion
* [PMM-7086](https://jira.percona.com/browse/PMM-7086): Re-mapped /prometheus/<end-point> to /victoriametrics/<end-point> but created aliases for users that still rely on the /prometheus/<end-point> in bookmarks and scripts (Thanks to Daniel Guzman Burgos for reporting this issue)
```

File: docs/release-notes/2.13.0.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+* [PMM-6713](https://jira.percona.com/browse/PMM-6713): Node Summary/Nodes Overview dashboards: Show External services on dashboards
```
Comment (shoffman-percona) [inline docs/release-notes/2.13.0.md:51]:
```suggestion
* [PMM-6713](https://jira.percona.com/browse/PMM-6713): Node Summary/Nodes Overview dashboards: External exporters can now be added to dashboard and shown as part of grouping of a broader service
```

File: docs/release-notes/2.13.0.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+* [PMM-7220](https://jira.percona.com/browse/PMM-7220): Keep consistent state of templates, rules, and alerts
```
Comment (shoffman-percona) [inline docs/release-notes/2.13.0.md:52]:
```suggestion
```

Comment (shoffman-percona) [reply docs/release-notes/2.13.0.md:52]: this is a first release of Int Alerting...so at best it was a defect fix and shouldn't be made public

File: docs/release-notes/2.13.0.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+* [PMM-7173](https://jira.percona.com/browse/PMM-7173): VictoriaMetrics updated to v1.50.2
```
Comment (shoffman-percona) [inline docs/release-notes/2.13.0.md:53]:
```suggestion
* [PMM-7173](https://jira.percona.com/browse/PMM-7173): VictoriaMetrics updated to v1.50.2: Includes HTML pages vs JSON output and new functions available for alerting rules [see all](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.50.2)
```

File: docs/release-notes/2.13.0.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+
+
+## Bugs Fixed
+
+* [PMM-7054](https://jira.percona.com/browse/PMM-7054): ProxySQL Instance Summary dashboard: no Node Metrics
+* [PMM-7092](https://jira.percona.com/browse/PMM-7092): PMM Server Docker update from 2.11.1 to 2.12.0 leaves container in unhealthy state (Thanks to Hubertus Krogmann for reporting this issue)
+* [PMM-7208](https://jira.percona.com/browse/PMM-7208): Confusing "Access denied" message for 'Viewer' users on many dashboards
+* [PMM-6987](https://jira.percona.com/browse/PMM-6987): No IP address shown in log file of OVF appliance running in headless mode
+* [PMM-7222](https://jira.percona.com/browse/PMM-7222): Fix filters configuration
```
Comment (shoffman-percona) [inline docs/release-notes/2.13.0.md:63]:
```suggestion
```

Comment (shoffman-percona) [reply docs/release-notes/2.13.0.md:63]: Defect, never was a released bug

**Review by shoffman-percona — COMMENTED** [review]
PTAL, tried to clarify the ones that made no sense to me and removed thigns that shouldn't be in RN

File: docs/release-notes/2.13.0.md (hunk 3 above)
Comment (PaulJacobs-percona) [reply docs/release-notes/2.13.0.md:52]: Will SKIP it

File: docs/release-notes/2.13.0.md (hunk 5 above)
Comment (PaulJacobs-percona) [reply docs/release-notes/2.13.0.md:63]: Which type should it be? @rnovikovP please change it IN JIRA ready for next jirabot run

Comment (PaulJacobs-percona) [reply docs/release-notes/2.13.0.md:63]: Was previously SKIP'd. Who unskipped it?

**Pull request merged**